<launch>
  <node name="euclidean_cluster" pkg="multi_kinect_euclidean_cluster" type="euclidean_cluster_node" output="screen">
    <param name="loop_rate" value="10" />
    <param name="source_pc_topic_name" value="/merged_cloud" />
    <param name="box_name" value="/clustering_result" />
    <param name="clustering_frame_id" value="world" />
  </node>
</launch>
